public/locales/ar/translation.json

```diff
@@ -166,8 +166,8 @@
       }
     ],
     "buttons": {
-      "opportunitiesBtn": "جميع الفرص",
-      "blogBtn": "جميع المدونات"
+      "opportunitiesBtn": "جميع الفرص {{arrow}}",
```

Comment:
you shouldnt need this here though. just append it

{t(opportunitiesBtn)} {arrow} in the html